core/src/index/knowhere/knowhere/index/vector_index/fpga/Apu.cpp

```diff
@@ -0,0 +1,252 @@
+
+#include <fstream>
+#include <chrono>
+#include "knowhere/index/vector_index/fpga/Apu.h"
+
+namespace Fpga {
+
+ApuInterface::ApuInterface (){
+    InitApu();
+}
+
+void
+ApuInterface::InitApu() {
+
+    gsi_prod_status_t status = gdl_init();
+
+    unsigned int num_existing_cards = 0 ;
+    unsigned int num_req_cards = 8;
+    status = gdl_context_count_get(&num_existing_cards);
+    std::cout << "gdl_context_count_get status is : " << status <<   " and num of cards is :  " << num_existing_cards << std::endl;
+
```

Comment:
Would you like to replace 'std::cout' with the log system?

Reply:
yes , we will change it to be align with all milvus debug message and prints .